client/js/components/procedure/admin/AdministrationPlaces.vue

```diff
@@ -56,6 +56,7 @@
           }" />
         <dp-button-row
           :busy="isLoading"
+          data-cy="addNewPlaceForm"
```

Comment:
`addNewPlace` would suffice i guess.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/3564/commits/50933cff9939c7d48f4618ceecaf819e46043a8e
Thanks!